backend/models/user.py

```diff
@@ -7,9 +7,13 @@ class User(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(64), index=True,
                       unique=True, nullable=False)
+    photo = db.Column(db.String(256), index=True,
```

Comment:
I don't think we should index on photos or make it unique either, people can definitely have the same profile pic